core/domain/rte_component_registry.py

```diff
@@ -37,9 +38,10 @@ class Registry(python_utils.OBJECT):
     def _refresh(cls):
         """Repopulate the registry."""
         cls._rte_components.clear()
-        with python_utils.open_file(
-            feconf.RTE_EXTENSIONS_DEFINITIONS_PATH, 'r') as f:
-            cls._rte_components = constants.parse_json_from_js(f)
+        package, filepath = os.path.split(
```

Comment:
Just to confirm, you tested manually that creating, modifying, etc. RTE components still works? (Would be good practice to put a video in the proof section of the PR description.)

Reply:
Yes, also this failed even if RTEs are not modified directly (I had to fix this just to get the Beam jobs working)